jena-arq/src/main/java/org/apache/jena/rdfs/DatasetGraphRDFS.java

```diff
@@ -67,6 +67,10 @@ public Graph getGraph(Node graphNode) {
         return new GraphRDFS(base, setup);
     }
 
+    @Override
```

Comment:
As I read the code on main:

`find()` ->  
`find()` in DatasetGraphWrapper ->  
default method DatasetGraph.find() ->  
find(Node.ANY, Node.ANY, Node.ANY, Node.ANY);

so this is not necessary.

And `Node.ANY` is better than null :smile:

Reply:
The default find() method of DatasetGraphWrapper goes to the delegated dataset which does not have the inferences - so DatasetGraphRDFS does have to override it to use findInf. I'll add a test case.

https://github.com/apache/jena/blob/29e8a5601da695741a342059ab9810794f65f7ca/jena-arq/src/main/java/org/apache/jena/sparql/core/DatasetGraphWrapper.java#L195-L197

> `Node.ANY`
Agreed.